src/index.js

```diff
@@ -71,7 +71,8 @@ app.get('/:id', function(req, res) {
 
   res.writeHead(200, {
     'Content-Type': 'application/zip',
-    'Content-disposition': 'attachment; filename="' + filename + '"'
+    'Content-disposition': 'attachment; filename="' + filename + '"',
+    'Content-Lenght':filename.size
```

Comment:
No, see line 77. We are piping the zipping process directly to the response stream.